videointelligence/setup.py

```diff
@@ -40,6 +40,7 @@
     install_requires=(
         'googleapis-common-protos >= 1.5.3, < 2.0dev',
         'google-gax >= 0.15.14, < 0.16dev',
+        'grpcio >= 1.2.0, < 1.6dev',
```

Comment:
@lukesneeringer Can you help me understand two things here (this is one of those auto-gen, flew by too fast, packages):

1. Does this package need `grpcio`? (If yes, I think we should explicitly pin here)
1. Why doesn't this package depend on `core`?